recipe-server/client/control/utils/apiFetch.js

```diff
@@ -35,13 +35,6 @@ export default function apiFetch(url, options = {}) {
     delete settings.data;
   }
 
-  return fetch(`${API_ROOT}${url}${queryString}`, settings)
-    .then(response => Promise.all([response.ok, response.json()]))
-    .catch(error => [false, { message: error.message }])
-    .then(([ok, data]) => {
-      if (!ok) {
-        throw data;
-      }
-      return data;
-    });
+  const response = await fetch(`${API_ROOT}${url}${queryString}`, settings);
+  return response.json();
```

Comment:
we want this to throw errors so we can catch them later:
https://github.com/mozilla/normandy/pull/704/files#diff-b2daff5c1e0fc92e2f54a8cd981c4d3aR29